### PR TITLE
Copyedit of "getting_started/step_by_step/scripting_continued"

### PR DESCRIPTION
**Pull request by gemkibeju**
Update typography and grammar for [getting_started/step_by_step/scripting_continued](https://godot.readthedocs.io/en/3.0/getting_started/step_by_step/scripting_continued.html).

I found myself more lost than the previous page. I feel this one focuses on providing a lot of information without providing explicit situations where the examples would be important to use. For that matter, I think including sections about physics processing and notifications when the user doesn't even know how to code in GDScript is a mistake.

I think it would be better to have a more direct tutorial that focuses on practical applications of GDScript. Of course this was mentioned in my last pull request #1095  , though this is the last scripting page in the getting_started section, so I might as well document my issues.

### CODE REVIEW

File: getting_started/step_by_step/scripting_continued.rst [hunk 1]
```diff
@@ -7,41 +7,41 @@ Processing
...
 
-However, it is still a very common case to have a script process on every
+However, it is still common to need a script to be processed on every
 frame. There are two types of processing: idle processing and physics
 processing.
 
-Idle processing is activated automatically when the method :ref:`Node._process() <class_Node__process>`
-is found in a script. It can be turned off (and back on) with the
+Idle processing is activated when the method :ref:`Node._process() <class_Node__process>`
+is found in a script. It can be turned and on with the
```
Comment (mhilbrunner) [inline getting_started/step_by_step/scripting_continued.rst]: on and off

File: getting_started/step_by_step/scripting_continued.rst [hunk 2]
```diff
@@ -7,41 +7,41 @@ Processing
...
 
 ::
 
     func _process(delta):
         # do something...
 
-The delta parameter describes the time elapsed (in seconds, as a
-floating point) since the previous call to "_process()".
+The delta parameter describes the time elapsed in seconds, as a
+floating point, since the previous call to ``_process()``.
```
Comment (mhilbrunner) [inline getting_started/step_by_step/scripting_continued.rst]: IMO: describes = contains, floating point number

File: getting_started/step_by_step/scripting_continued.rst [hunk 3]
```diff
@@ -119,14 +117,14 @@ function in your script:
...
-shows the notifications it can receive. However, for most cases GDScript
+shows the notifications it can receive. However, in most cases GDScript
 provides simpler overrideable functions.
 
 Overrideable functions
 ----------------------
 
-Nodes provide many useful overrideable functions, which are described as
-follows:
+Such overrideable, which are described as
```
Comment (mhilbrunner) [inline getting_started/step_by_step/scripting_continued.rst]: missing "functions"

**Comment by mhilbrunner** [comment]
Ah, seems you already fixed two of those in the second commit. There are some conflicts with master, would be great if you could fix them :)

**Comment by gemkibeju** [comment]
@mhilbrunner That should do it.

**Pull request merged**